_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +783,1636 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
-    "fee": 290000000000,
-    "id_hash": "11d4cff23e610fac6a2b89187ad61d429a5e226652693dcac5d83d506eb92b96",
-    "kept_by_block": false,
-    "last_failed_height": 0,
-    "last_failed_id_hash": "0000000000000000000000000000000000000000000000000000000000000000",
-    "max_used_block_height": 954508,
-    "max_used_block_id_hash": "03f96b374778bc059e47b96e2beec2e6d4d9e0ad39afeabdbcd77e1bd5a62f81",
-    "receive_time": 1457676127,
-    "tx_json": "{\n  \"version\": 1, \n  \"unlock_time\": 0, \n  \"vin\": [ {\n      \"key\": {\n        \"amount\": 70000000000, \n        \"key_offsets\": [ 63408, 18978, 78357, 16560\n        ], \n        \"k_image\": \"7319134bfc50668251f5b899c66b005805ee255c136f0e1cecbb0f3a912e09d4\"\n      }\n    },  ...  ], \n  \"vout\": [ {\n      \"amount\": 80000000000, \n      \"target\": {\n        \"key\": \"094e6a1b187385533665f89db741149f42d95fdc50bdd2a2384bcd1dc5209c55\"\n      }\n    },  ...  ], \n  \"extra\": [ 2, 33, 0, 15, 56, 190, 21, 169, 77, 13, 182, 209, 51, 35, 54, 96, 89, 237, 96, 23, 24, 107, 240, 79, 40, 86, 64, 68, 45, 166, 119, 192, 17, 225, 23, 1, 31, 159, 145, 15, 173, 255, 165, 192, 55, 84, 127, 154, 163, 25, 85, 204, 212, 127, 147, 133, 118, 218, 166, 52, 78, 188, 131, 235, 9, 159, 105, 158\n  ], \n  \"signatures\": [ \"966e5a67fbdbf72d7dc0364b705121a58e0ced7e2ab45747b6b154c05a1afe04fac4aac7f64faa2dc6dd4d51b8277f11e2f2ec7729fac225088befe3b8399c0b71a4cb55b9d0e20f93d305c78cebceff1bcfcfaf225428dfcfaaec630c88720ab65bf5d3399dd1ac82ea0ecf308b3f80d9780af7742fb157692cd60515a7e2086878f082117fa80fff3d257de7d3a2e9cc8b3472ef4a5e545d90e1159523a60f38d16cece783579627124776813334bdb2a2df4171ef1fa12bf415da338ce5085c01e7a715638ef5505aebec06a0625aaa72d13839838f7d4f981673c8f05f08408e8b372f900af7227c49cfb1e1febab6c07dd42b7c26f921cf010832841205\",  ...  ]\n}"
-  },
-  ...]
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "status": "OK"
+  }
 }
 ```
 
 
-### **/stop_daemon**
+### **sync_info**
 
-Send a command to the daemon to safely disconnect and shut down.
+Get synchronisation informations
+
+Alias: *None*.
 
 Inputs: *None*.
 
 Outputs:
 
+* *height* - unsigned int;
+* *peers* - array of peer structure, defined as follows:
+  * *info* - structure of connection info, as defined in [get_connections](#get_connections)
+* *spans* - array of span structure, defined as follows (optional, absent if node is fully synced):
+  * *connection_id* - string; Id of connection
+  * *nblocks* - unsigned int; amount of blocks being synced
+  * *rate* - unsigned int; connection rate
+  * *remote_address* - string; peer address the node is spanning from
+  * *size* - unsigned int; synchronisation size
+  * *speed* - unsigned int; connection speed
+  * *start_block_height* - unsigned int; starting block height being synchronised
 * *status* - string; General RPC error code. "OK" means everything looks good.
+* *target_height* - unsigned int; target height the node is syncing to (optional, absent if node is fully synced)
```

Comment:
Most of those just paraphrase the parameter name without making much sense. If you don't know what some field is, you can ask.
nblocks: number of blocks in that span
size: total number of bytes in that span's blocks (including txes)
start_block_height: block height of the first block in that span
remote_address: ...the node is downloading (or has downloaded) than span from

Reply:
ok. I should have asked you're right.